src/App.vue

```diff
@@ -410,13 +428,13 @@
               <button
                 type="button"
                 :class="{
-                  'btn-primary': selectedTags.length < 4,
-                  'btn-secondary': selectedTags.length >= 4,
+                  'btn-primary': selectedTags.length < 3,
```

Comment:
ok like this but that was the good opportunity to set a `const MinNBTags=3;`

Reply:
Right and I do the same for the max nb tags